product_docs/docs/postgres_distributed_for_kubernetes/1/ldap.mdx

```diff
@@ -31,10 +31,10 @@ spec:
           suffix: ",dc=example,dc=org"
 ```
 
-Here is a example of LDAP configuration using `search+bind` mode in PGDGroup. 
-In this mode, the postgres is first bound to the LDAP using `bindDN` with its password stored
-in the secret `bindPassword`, then postgres tries to perform a search under `baseDN` to find a 
-username that matches the item specified by `searchAttribute`, if a match is found, postgres finally 
+This example shows configuring LDAP using `search+bind` mode in PGDGroup.
+In this mode, the postgres database is first bound to the LDAP using `bindDN` with its password stored
+in the secret `bindPassword`. Then Postgres tries to perform a search under `baseDN` to find a 
+username that matches the item specified by `searchAttribute`. If a match is found, Postgres finally 
 verifies the entry and the password against the LDAP server.
```

Comment:
Please check the above clarifications of the word "postgres." it looked like the first one might have meant the name of a database and the others referred to the product, but that needs to be checked.

Reply:
I think your edit goes in the right direction, but looking at [Postgres LDAP docs](https://www.postgresql.org/docs/current/auth-ldap.html), it seems like the "binding" is done per server, not per database, so I would suggest changing "postgres database" to "Postgres database server".
@jsilvela to confirm

Reply:
Agree with Gaby's comment, but also, "server", like "database", is already pretty overloaded.
In the context the Kubernetes products we tend to talk more about instances and pods.
Like the other question about the "errant Postgres", this is meant to call out that the actual work is performed by Postgres.
I know I just used another errant Postgres, I know.
In other parts of the CNPG / PG4K documentation we distinguish operator from operand.
Might it be useful here? I.e. the bind is performed at the operand level.

Reply:
@ebgitelman  I think in that case we have two options: 

- We can talk about Postgres as the technology performing these kind of operations. If we want to remain general, we could say that the "Postgres instance tries to perform a search under ...". 
- We can be specific and say that the operand is the entity performing the binding and link to https://www.enterprisedb.com/docs/postgres_distributed_for_kubernetes/latest/identify_images/identify_image_name/#operand-image-name. Although this is technically more accurate, I am not sure we would be introducing more terms here that are potentially confusing without bringing much value. 

WDYT?

Reply:
Let's go with the first option.

Reply:
Please check edit and if there's something else you want, please let me know the correct wording.